src/shared/symbols.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+import Symbol from 'es-symbol'
+
+export const ACTION_HANDLER = Symbol('action creator handler')
```

Comment:
Hmm, I can't think of any other way to get around this since they are used by most parts of the app.

It might not actually be that bad of a thing because the built files in `dist/` won't give users direct access to these symbols anyway.  So the Symbols are exposed to the src only, but not to the end user.